runtimes/runtimes/lsp/router/lspRouter.ts

```diff
@@ -32,11 +34,34 @@ export class LspRouter {
         lspConnection.onRequest(getConfigurationFromServerRequestType, this.handleGetConfigurationFromServer)
     }
 
+    public get clientSupportsShowNotification() {
+        return (
+            this.clientInitializeParams?.initializationOptions?.aws.awsClientCapabilities?.window?.showNotification ??
+            false
+        )
+    }
+
     initialize = async (
         params: InitializeParams,
         token: CancellationToken
     ): Promise<InitializeResult | ResponseError<InitializeError>> => {
         this.clientInitializeParams = params
+
+        let responsesList = await Promise.all(this.servers.map(s => s.initialize(params, token)))
+        responsesList = responsesList.filter(r => r != undefined)
+        const errors = responsesList.find(el => el instanceof ResponseError)
```

Comment:
supernit: `.find` returns single element, singular `error` as name is more correct